### PR TITLE
Initial Container Cost

### PR DESCRIPTION
**Pull request by truclk**

### CODE REVIEW

File: nops_k8s_agent/nops_k8s_agent/container_cost/base_prom.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+class BaseProm:
+    def __init__(self):
+        if settings.NOPS_K8S_AGENT_PROM_TOKEN:
+            headers = {"Authorization": settings.NOPS_K8S_AGENT_PROM_TOKEN}
+        else:
+            headers = {}
+        # self.prom_client = PrometheusConnect(url=settings.PROMETHEUS_SERVER_ENDPOINT, headers=headers, disable_ssl=True)
+        self.prom_client = PrometheusConnect(
+            url="http://prometheus-server.prometheus-system.svc.cluster.local:80", headers=headers, disable_ssl=True
```
Comment (andremilk) [inline nops_k8s_agent/nops_k8s_agent/container_cost/base_prom.py:17]: Can't we stick to using settings.PROMETHEUS_SERVER_ENDPOINT?

File: nops_k8s_agent/nops_k8s_agent/container_cost/base_metrics.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        group_by_list = self.list_of_metrics.get(metric_name)
+        group_by_str = ",".join(group_by_list)
+        now = datetime.now(pytz.utc)
+
+        if period == "last_hour":
+            start_time = now.replace(minute=0, second=0, microsecond=0) - timedelta(hours=1)
+            end_time = start_time + timedelta(hours=1) - timedelta(seconds=1)
+        elif period == "last_day":
+            start_time = now.replace(hour=0, minute=0, second=0, microsecond=0) - timedelta(days=1)
+            end_time = start_time + timedelta(days=1) - timedelta(seconds=1)
```
Comment (andremilk) [inline nops_k8s_agent/nops_k8s_agent/container_cost/base_metrics.py:31]: We can get this line outside the if statements and remove a duplicate line

**Review by andremilk — CHANGES_REQUESTED** [review]
Minor change that are not actually a show stopper as this is an initial work on this.
I just wanted to point those out

File: nops_k8s_agent/nops_k8s_agent/container_cost/base_prom.py (hunk 1 above)
Comment (truclk) [reply nops_k8s_agent/nops_k8s_agent/container_cost/base_prom.py:17]: Fixed it

File: nops_k8s_agent/nops_k8s_agent/container_cost/base_metrics.py (hunk 2 above)
Comment (andremilk) [reply nops_k8s_agent/nops_k8s_agent/container_cost/base_metrics.py:31]: Actually I read that wrong, you are right about this

**Review by andremilk — APPROVED** [review]

**Pull request merged**